glitchdotcom/Glitch-Community
# Stop the continual restarting!

## Links
* https://glitch.com/edit/#!/search-seo?

## Changes:
* we'd switched to `pm2-runtime` from simply `pm2` in the `start` script at some point since I started work. I didn't catch that change. Updating that in the `start` and new `stop` scripts seems to resolve the continual restarting Cassey observed in her first remix after pushing the new script to `master`
* I also made a change to `local-deploy.sh` to try and resolve a different issue, but that ended up not being the problem, so this reverts that (`staging`-only)change.

## How To Test:
* Remix community
* `git pull origin staging` (will get you these changes _before_ they're merged into `master`)
* wait for rebuild, check logs to make sure you see the expected output and that the project isn't just restarting over and over.

## Feedback I'm looking for:
* does this fix the _immediate_ problem?
